src/coresyftools/coresyftools/tool.py

```diff
@@ -38,9 +59,27 @@ def __init__(self, run_script_file_name):
             self.context_directory, self.MANIFEST_FILE_NAME)
         self.manifest = get_manifest(self.manifest_file_name)
         self.arg_parser = CoReSyFArgumentParser(self.manifest)
+        if 'command' in self.manifest:
+            self._validate_command(self.manifest['command'])
         self.operation = self.manifest.get('operation', {})
         self._validate_operation(self.operation)
 
+    def _validate_command(self, command):
+        placeholders = self._extract_command_placeholders(command)
+        for placeholder in placeholders:
+            if placeholder not in self.arg_parser.identifiers:
+                raise UnexpectedCommandPlaceholder(placeholder)
+        for identifier in self.arg_parser.identifiers:
+            if identifier not in placeholders:
+                raise MissingCommandPlaceholderForOption(identifier)
+
+    def _extract_command_placeholders(self, command_template):
```

Comment:
Too many lines (linting)

Reply:
I meant columns (I always say lines!). But you are right, I don't know why I saw this. Ignore.